cni/pkg/nodeagent/server.go

```diff
@@ -130,6 +180,7 @@ func NewServer(ctx context.Context, ready *atomic.Value, pluginSocket string, ar
 			netServer:          netServer,
 			hostIptables:       hostIptables,
 			hostsideProbeIPSet: set,
+			hostNetNS:          hostNetNS,
```

Comment:
If we can always `stat` this from our `proc` mount, then we don't need to carry a live FD around as state.